test_api.py

```diff
@@ -0,0 +1,128 @@
+import requests
+import sqlite3
+
+from datetime import datetime
+# from pprint import pprint as pp
+
+
+DB = 'test.db'
+ID_G = iter(range(1, 10_000))
+
+init_data = [
+    (next(ID_G), 'Prepare a dinner for a week', 'home',
+     datetime.today().strftime('%d-%m-%y'), 0, ''),
+    (next(ID_G), 'Make the report of the week', 'job',
+     datetime.today().replace(day=datetime.today().day + 2).strftime('%d-%m-%y'),
+     0, ''),
+    (next(ID_G), 'Make a homework for lesson 10', 'studies',
+     datetime.today().replace(day=datetime.today().day + 1).strftime('%d-%m-%y'),
+     0, ''),
+    (next(ID_G), 'Read the documentation about descriptors', 'studies',
+     datetime.today().strftime('%d-%m-%y'), 0, '')
+]
+
+
+def test_add_new_task():
+    print('test POST')
+    new_task = {
+        'id': next(ID_G),
+        'title': 'Finish task 4',
+        'category': 'job',
+        'expired_date': datetime.today().replace(day=datetime.today().day + 2).strftime('%d-%m-%y'),
+    }
+
+    response = requests.post('http://localhost:5000/api/tasks/', json=new_task)
+    # print(response.content)
```

Comment:
Could you remove temporal comments and print statements?

Reply:
Removed.